src/test/java/software/amazon/event/kafkaconnector/EventBridgeSinkConnectorIT.java

```diff
@@ -252,7 +87,9 @@ public void sendJsonRecordToKafkaReceiveFromSQS() {
     }
     log.info("successfully sent json test record to kafka");
 
-    var receivedMessage = new AtomicReference<Message>(null);
+    s3client.putObject(
```

Comment:
IMHO this should be done after receiving from SQS to avoid races and endure START STOP sequence does not contain offloaded S3 keys in between